Src/driver.c

```diff
@@ -89,8 +101,17 @@ static input_signal_t inputpin[] = {
 #endif
 // Limit input pins must be consecutive in this array
     { .id = Input_LimitX,         .port = X_LIMIT_PORT,       .pin = X_LIMIT_PIN,         .group = PinGroup_Limit },
+#ifdef X2_LIMIT_PIN
+    { .id = Input_LimitX_Max,     .port = X2_LIMIT_PORT,      .pin = X2_LIMIT_PIN,        .group = PinGroup_Limit },
+#endif
```

Comment:
Hmm, I have to add new definitions? Using *_Max for the second min pin is ambiguous. Have to think about this as I have changed the core to support up to four limit switches per axis even though no more than three is ever going to be assigned. 2 min switches and one max switch.

Reply:
To be honest I based this on something i found in one of the other drivers. not even sure i really need it....

Reply:
This was more a comment to myself and/or for discussion. I have added definitions for a secondary min input in crossbar.h: `Input_LimitX_2` etc. to resolve the ambiguity (not yet committed). I'll fix the driver code later so you do not have to worry about it.